bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/AvailabilityTracker.java

```diff
@@ -33,6 +34,21 @@ public interface AvailabilityTracker {
      */
     public void addAvailabilityTopic(String availability_topic, String payload_available, String payload_not_available);
 
+    /**
+     * Adds an availability topic to determine the availability of a device.
+     * <p>
+     * Availability topics are usually set by the device as LWT.
+     *
+     * @param availability_topic
```

Comment:
If you add the javadoc, can you please do it fully and so set a description to each parameter.

Reply:
Done